packages/blocks/divider/src/webpack-dev-server.tsx

```diff
@@ -4,10 +4,10 @@
  */
 import React from "react";
 import ReactDOM from "react-dom";
-import Component from "./index.ts";
+import Component from "./index";
 
 const node = document.getElementById("app");
 
-const App = () => <Component />;
+const App = () => <Component color="red" height="2px" />;
```

Comment:
Rename this file from `.js` to `.tsx` to get autocompletions.

Reply:
Makes sense. This is here just to showcase the `color` capabilities of the block though.